POP.nix

```
@@ -205,6 +211,21 @@
            composeProto (extensionProto g) (extensionProto f)
    */
 
+  # Build a set of functions that return a pop thats extended by the extender function
+  # buildExtenders: (Attrset (_: Extension A B)) C D -> (Attrset (_: Pop A B))
+  buildExtenders = extenders: self: super:
+    lib.mapAttrs (
+      name: extender: (
+        arg:
+          pop {
+            name = self.__meta__.name;
+            supers = [self];
+            extension = extender arg;
+          }
```

```suggestion
         extender arg self super
```
So extenders can return a pop that extends self, or they can return anything else that could, optionally, depend on self.